QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002174	$Epitaph of Juan
+QUEST_20150714_002175	$I lie here without yet creating thirteen thousand pieces of armor. {nl}I hope to finish them beside the Goddesses.
+QUEST_20150714_002176	$I can hear a whisper in my head.
+QUEST_20150714_002177	$Please install the device I gave you in the 2nd district in Crystal Mine.
+QUEST_20150714_002178	$The location is Sidabro Rest Place in Veya Valley. {nl}I hope he's still alive.
+QUEST_20150714_002179	$Klaipeda Market
+QUEST_20150714_002180	$Give me the effigy. I shall calm the spirits.
+QUEST_20150714_002181	$The monsters will feel threatened by the energies that spurt out from the crystals containing my authority and begin their attack.
+QUEST_20150714_002182	$Please protect the crystals so that they may produce my authority and spread it to Sirdgelos Forest.
+QUEST_20150714_002183	$You prepared this, brother? You're truly a great man.
+QUEST_20150714_002184	$I can sense the wrath of the spirits.
+QUEST_20150714_002185	$Incense Burner
+QUEST_20150714_002186	$The scent of the Incense Burner is spreading.
+QUEST_20150714_002187	$I can feel the spirits are complaining.
+QUEST_20150714_002188	$I can feel the pain of the spirits.
+QUEST_20150714_002189	$I can feel the regrets of the spirits.
+QUEST_20150714_002190	$The seal is not yet ready? {nl}It pains me so that you are the only one I can trust.
+QUEST_20150714_002191	$Can you solve the complaints of the villagers' spirits in Seven Valley?
+QUEST_20150714_002192	$The Basics of Ignition
+QUEST_20150714_002193	$Ignition differs by elements, but with the blessing from the Goddess, there may be cases where we can ignore that. {nl}To do so, we should try hard to achieve the scholarly knowledge and the faithfulness to the Goddesses.
+QUEST_20150714_002194	$1. Dried environment{nl}2. Something that can be ridden on. {nl}3. Enough mana. {nl}4. The blessing of the Goddesses.
+QUEST_20150714_002195	$Flurry… turned into a handful of ashes by illuminating a great light. {nl}Did that demon do something? {nl}I had no idea, even as a probationary wizard.
+QUEST_20150714_002196	$[Caution]{nl}- It is the last stage of the experiment. Be cautious when handling it -
+QUEST_20150714_002197	$1. Do not overreact Schilt Essences into a sample of Petrification Detector. {nl}2. Do not keep the Schilt Amplifier activated for a long time. {nl}3. Even if you become immune to petrification using the Schilt Amplifier, it will only be temporary so do not stay in the lab room for too long.
+QUEST_20150714_002198	$[Usage Limit]{nl}- Due to a recent incident, we sealed the isolated lab room. {nl}Only people permitted by the chief can enter the room via the portal.
+QUEST_20150714_002199	$[Method of Usage]{nl}1. Use the Schilt Essences on the portal generator. {nl}2. One person can enter after a certain number of Essences are used. {nl}3. When more than one person enters simultaneously, please notify and ask the chief.
```

Comment:
3. seems to imply you have to notify and ask the chief in case you want to enter with someone. Check with a translator, please.